addons/faster-project-loading/addon.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Faster project loading",
+  "description": "Makes project loading faster when opening directly to the editor.",
```

Comment:
Maybe the description could be improved.

Reply:
How about this?
```suggestion
  "description": "Fixes a Scratch bug that slows down project loading if the editor is opened from a link.",
```